resources/k8s-artifacts/choreo-connect/config-toml-configmap.yaml

```diff
@@ -61,6 +66,8 @@ data:
       securedListenerHost = "0.0.0.0"
       securedListenerPort = 9095
       clusterTimeoutInSeconds = 20
+      # system hostname for system API resources (eg: /testkey and /health)
+      systemHost = "localhost"
```

Comment:
Can you please add the same config for the eventhub enabled config map as well

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1899/commits/c833924d5280a2a0b8d734ddea208dad35570dc5